package-lock.json

```diff
@@ -15251,6 +15264,16 @@
             "uuid": "^3.3.2"
           },
           "dependencies": {
+            "form-data": {
```

Comment:
This is a nested dependency of `cypress` which uses `request` somewhere under the hood. Really just npm doing dedupe work.